app/css/main.css

```diff
@@ -329,13 +357,25 @@ menu ul li.selected {
 }
 
 .center input, .center select {
-    text-align:center;
+    text-align:left;
```

Comment:
if it's text-align left, it shouldn't be called the center class. Maybe it's better to remove the .center class to the element.